modules/ROOT/pages/telemetry-exporter.adoc

```diff
@@ -0,0 +1,146 @@
+= Export Anypoint Platform Data to Third-Party Apps
+
+The Telemetry Exporter feature enables you to export xref:access-management::audit-logging.adoc[_audit logs_] to third-party observability and analytics apps, such as Azure Monitor, Datadog, and Splunk HEC.
```

Comment:
Do you have to keep calling this a feature? Can you just say "Telemetry Exporter enables you to..."?

Reply:
If I want to capitalize it, which I do, yes. Unless we removed that rule.

Reply:
```suggestion
The Telemetry Exporter enables you to export xref:access-management::audit-logging.adoc[_audit logs_] to third-party observability and analytics apps, such as Azure Monitor, Datadog, and Splunk HEC.
```